base.c

```diff
@@ -168,8 +168,27 @@ int isOneByteOp() {
         (token >= ADD && token <= MODULO);
 }
 
-int isOp() {
-  return isOneByteOp();
+int isKonstantOp(){
+    //TODO: implement isKonstantOp()
+    return false;
 }
 
+int isVarOp(){
+    //TODO: implement isVarOp()
+    return false;
+}
 
+int isOp() {
+  if(isOneByteOp()){
+      return 0;
```

Comment:
what if instead of returning raw numbers we add some #define statements in the header, and we check against those in the switch statement?